primo/data_parser/input_options_well_data.py

```diff
@@ -0,0 +1,214 @@
+# Reduction Program (MERP) and National Energy Technology Laboratory's (NETL)
+# National Emissions Reduction Initiative (NEMRI).
+#
+# NOTICE. This Software was developed under funding from the U.S. Government
+# and the U.S. Government consequently retains certain rights. As such, the
+# U.S. Government has been granted for itself and others acting on its behalf
+# a paid-up, nonexclusive, irrevocable, worldwide license in the Software to
+# reproduce, distribute copies to the public, prepare derivative works, and
+# perform publicly and display publicly, and to permit others to do so.
+#################################################################################
+
+"""
+This module defines all the input options for objects of WellData class
+"""
+
+
+# Installed libs
+from pyomo.common.config import (
+    Bool,
+    ConfigDict,
+    ConfigValue,
+    In,
+    NonNegativeFloat,
+    NonNegativeInt,
+)
+
+# User libs
+from primo.utils.domain_validators import InRange
+from primo.data_parser.default_data import SUPP_IMPACT_METRICS
+
+
+# This dictionary contains default values to fill with,
+# if the data is not provided.
+binary_type_metrics = {
+    "leak": True,  # Well is leaking
+    "compliance": True,  # Well is compliant
+    "violation": False,  # Well is not in violation
+    "incident": False,  # Incident is assumed to be False
+    "h2s_leak": False,  # H2S Leak is absent
+    "brine_leak": False,  # Brine leak is absent
+    "buildings_near": False,  # No buildings nearby
+    "buildings_far": False,  # No distant buildings
+    "fed_wetlands_near": False,  # No federal wetlands nearby
+    "fed_wetlands_far": False,  # No distant federal wetlands
+    "state_wetlands_near": False,  # No state wetlands nearby
+    "state_wetlands_far": False,  # No distant state wetlands
+    # Assuming that well integrity is good, if it is not provided
+    "well_integrity": False,
+}
+
+# Input options for well data class
+CONFIG = ConfigDict()
+CONFIG.declare(
+    "census_year",
+    ConfigValue(
+        default=2020,
+        domain=In(list(range(2020, 2101, 10))),
+        doc="Year for collecting census data",
+    ),
+)
+CONFIG.declare(
+    "ignore_operator_name",
+    ConfigValue(
+        default=True,
+        domain=Bool,
+        doc="Remove well if operator name is not provided",
+    ),
+)
+CONFIG.declare(
+    "missing_age",
+    ConfigValue(
+        default="fill",
+        domain=In(["fill", "estimate", "remove"]),
+        doc="Method for processing missing age information",
+    ),
+)
+CONFIG.declare(
+    "missing_depth",
+    ConfigValue(
+        default="fill",
+        domain=In(["fill", "estimate", "remove"]),
+        doc="Method for processing missing depth information",
+    ),
+)
+CONFIG.declare(
+    "fill_age",
+    ConfigValue(
+        default=100,
+        # Assuming that no well is older than 250 years
+        domain=InRange(0, 250),
```

Comment:
Minor but maybe we should change the range to 350 since we are hoping to support this through 2101 per line 59 :-)